care/facility/api/serializers/patient_consultation.py

```diff
@@ -393,6 +393,9 @@ class PatientConsultationDischargeSerializer(serializers.ModelSerializer):
     discharge_reason = serializers.ChoiceField(
         choices=DISCHARGE_REASON_CHOICES, required=True
     )
+    referred_to_external = serializers.CharField(
+        required=False, allow_null=True, allow_blank=True
+    )
```

Comment:
Explicitly specifying this is not required

Reply:
@sainak I have pushed the suggested changes. Kindly review it.